Challenge reminder email (#173)
srobo/team-emails: SR2024/2023-12-13-challenge-reminder.md

> Hi $TODO_CONTACT_GIVEN_NAME,

Reviewer: Missing header? What's the subject?

> Just a reminder that the first challenge submission deadline passed over the weekend. 

Reviewer: This opening sentence feels confusing to me. Starting out as a reminder sounds like it's going to be something the teams may need to do something about, yet it being in the past negates that.

Perhaps opening with the "Congratulations" line would be better?

```suggestion
Congratulations to the 15 teams that completed one of the challenges in time for the first submission deadline this past weekend. We've been really impressed with the challenge submissions this year, it's been great to see the creativity of so many teams so early on in the competition.
```

(And then obviously drop that line from below)

> If you aren't aware of the challenges, you can find out more about them in the rulebook [here](https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges)

Reviewer: Please can we avoid using "here" as a link text? It's generally regarded as unclear for users (especially those using assistive technologies) as it provides little description of what the link _is_. Instead we should prefer to use text which describes the target as the link.

```suggestion
If you aren't aware of the challenges, you can find out more about them in the [rulebook](https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges).
```

Congratulations to the 15 teams that managed to complete a challenge by that deadline.

> The next challenge deadline is 20th January 2024, so remember time will be short when your team gets back from Christmas break. There are another 5 league points up for grabs for anyone who manages to submit a challenge for this deadline.

Reviewer: ```suggestion
The next [challenge deadline](https://studentrobotics.org/events/sr2024/second-challenge-submission-deadline/) is 20th January 2024, so remember time will be short when your team gets back from Christmas break. There are another 5 league points up for grabs for anyone who manages to submit a challenge for this deadline.
```

> Your team can re-submit if their first attempt didn't pass, if you have any questions, just ask us via e-mail or Discord.

Reviewer: This feels like it's two sentences to me?
```suggestion
Your team can re-submit if their first attempt didn't pass. If you have any questions, just ask us via e-mail or Discord.
```
It also feels like a bit more context might be useful?
```suggestion
Your team can re-submit if their attempts aren't judged to have passed the challenge, so it's worth having a go in plenty of time before the deadline. If you have any questions, just ask us via e-mail or Discord.
```

---
to: Student Robotics 2024
subject: Student Robotics Challenge submissions
---

Hi $TODO_CONTACT_GIVEN_NAME,

We hope you had an excellent new year and holiday break.

Congratulations to the 15 teams that completed one of the challenges in time for the first submission deadline in December. We've been really impressed with the challenge submissions this year, it's been great to see the creativity of so many teams so early on in the competition.

If you aren't aware of the challenges, you can find out more about them in the [rulebook](https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges)

The next [challenge deadline](https://studentrobotics.org/events/sr2024/second-challenge-submission-deadline/) is 20th January 2024, so remember time will be short when your team gets back from Christmas break. There are another 5 league points up for grabs for anyone who manages to submit a challenge for this deadline.

Your team can re-submit if their attempts aren't judged to have passed the challenge, so it's worth having a go in plenty of time before the deadline. If you have any questions, just ask us via e-mail or Discord.

> Thanks, $TODO_VOLUNTEER_NAME

Reviewer: Just a though -- I think we'd normally sign off with like "-- The Competition Team" for general mailings.

---
to: Student Robotics 2024
subject: Student Robotics Challenge submissions
---

Hi $TODO_CONTACT_GIVEN_NAME,

We hope you had an excellent new year and holiday break.

> Congratulations to the 15 teams that completed one of the challenges in time for the first submission deadline this past weekend. We've been really impressed with the challenge submissions this year, it's been great to see the creativity of so many teams so early on in the competition.

Reviewer: ```suggestion
Congratulations to the 15 teams that completed one of the challenges in time for the first submission deadline in December. We've been really impressed with the challenge submissions this year, it's been great to see the creativity of so many teams so early on in the competition.
```

If you aren't aware of the challenges, you can find out more about them in the [rulebook](https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges)

> 
> Congratulations to the 15 teams that managed to complete a challenge by that deadline.

Reviewer: This line duplicates the first sentence of the second paragraph. Is that expected? (it feels a bit redundant to me -- perhaps just drop this line?)
```suggestion
```
